src/auth/session/storage/redis.ts

```diff
@@ -105,9 +106,22 @@ export class RedisSessionStorage implements SessionStorage {
 
   private async init() {
     this.client = createClient({
-      ...this.options,
+      socket: this.options.socket,
+      username: this.options.username,
+      password: this.options.password,
+      name: this.options.name,
+      database: this.options.database,
+      commandsQueueMaxLength: this.options.commandsQueueMaxLength,
+      disableOfflineQueue: this.options.disableOfflineQueue,
+      readonly: this.options.disableOfflineQueue,
+      legacyMode: this.options.legacyMode,
+      isolationPoolOptions: this.options.isolationPoolOptions,
```

Comment:
Thank you! Definitely, using spread operator makes it much easier to read. 

I was having troubles with making types work, because of a known Redis issue regarding `RedisClientType`. I added a solution towards it which was [previously discussed in their repo](https://github.com/redis/node-redis/issues/1673).